foglamp.version

```diff
@@ -1 +1 @@
-foglamp_version>=1.6
+foglamp_version>=1.7
```

Comment:
@Singhal-Vaibhav Missing service_notification.version. Please add.

Here we have foglamp.version as well, is this really required @pintomax @MarkRiddoch
I think we are NOT keeping `foglamp.version` file in `rule-` plugins; as direct dependency is `notification service` which has dependency  check for foglamp version

Reply:
@praveen-garg we need foglamp.version for any plugin

Reply:
Having foglamp version dependency allows the plugin to express its dependency, that could be different from the notification service, so no issue with this.